resources/packs/azure/azure_cloud_defender.go

```diff
@@ -0,0 +1,138 @@
+package azure
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	security "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/security/armsecurity"
+	"go.mondoo.com/cnquery/resources/packs/core"
+)
+
+func (a *mqlAzureCloudDefender) id() (string, error) {
+	return "azure.cloudDefender", nil
```

Comment:
going forward we need to scope this to a subscription

Reply:
Good point. I think currently we do not support multiple subscriptions on the microsoft provider, so I reckon for now it's okay

Reply:
That will need to change within the next weeks since we need full asset hierarchy for Azure as well.